src/service/filtering/filters.py

```diff
@@ -92,6 +95,68 @@ def _gt(num: int, min_: int, name: str):
     return num
 
 
+class BooleanFilter(AbstractFilter):
```

Comment:
it occurs to me that for heatmaps we only need to use filters if there's a search on the display name. Otherwise we could probably just use regular AQL. I don't really see a strong reason to avoid arangosearch though, other than it's eventually consistent

Reply:
I'd like to keep things consistent. It's more easier to maintain.

Reply:
:+1: